spec/models/retirement_manager_spec.rb

```diff
@@ -0,0 +1,33 @@
+describe RetirementManager do
+  describe "#check" do
+    let(:ems) { FactoryGirl.create(:ext_management_system) }
+    let(:zone) { FactoryGirl.create(:zone, :name => "api_zone", :ext_management_systems => [ems]) }
+    let(:my_server) { FactoryGirl.create(:miq_server, :zone => zone) }
```

Comment:
I think you can use `EvmSpecHelper.local_miq_server` and remove the `let(:zone)`